components/ProgressBar.js

```diff
@@ -1,8 +1,84 @@
-import React, { useMemo } from "react";
-import { useSupabaseDonations } from "../hooks/useSupabaseDonations";
+import React, { useMemo, useState } from 'react';
+// import { useSupabaseDonations } from '../hooks/useSupabaseDonations';
+import { useTeamSeasDonations } from '../hooks/useTeamSeasDonations';
+import styles from '../styles/ProgressBar.module.css';
 
 export default function ProgressBar() {
-  const { status, data, error, isFetching } = useSupabaseDonations();
+  // const { status, data, error, isFetching } = useSupabaseDonations();
+  const { data } = useTeamSeasDonations();
```

Comment:
Using data directly from the Team Seas API - Supabase didn't seem to update whenever I added another donation?